src/test/resources/pages/instructorCommentsPageSearchNormal.html

```diff
@@ -197,11 +197,11 @@
                     <b>
                       From:
                     </b>
-                    student1 In Course1\u003c/option\u003e\u003c/td\u003e\u003c/div\u003e\u0027\" (Team 1.1\u003c/td\u003e\u003c/div\u003e\u0027\")
```

Comment:
Nope, nothing to do with it. This is *wrong* sanitization: sanitizing to unicode is not the correct way for web browsers; instead it should be sanitized for HTML display. Thus this also serves as a bug fix.
Proof:
![screen shot 2016-12-09 at 9 14 20 am](https://cloud.githubusercontent.com/assets/7261051/21036646/72476d80-be02-11e6-8ada-e5b773b2c5e4.png)